manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml

```diff
@@ -44,7 +44,7 @@ spec:
           description: CheInstallationSpec defines the desired state of CheInstallation
           properties:
             cheOperatorSpec:
-              description: The configuration required for che operator
+              description: The configuration required for Che operator
```

Comment:
@xcoulon Please, do not change anyting in the released manifests - they are already released - we cannot modify anything there.

Reply:
reverting in https://github.com/codeready-toolchain/toolchain-operator/pull/56

Reply:
Good catch @MatousJobanek

Reply:
ah right, my apologies! 🤦‍♂